### ISSUE TITLE
New genetics UI is broken for some people

### ISSUE THREAD

**Issue by Kingtrin** [tgstation/tgstation#46660]
## Reproduction: Attempt to use the genetics console. Have it fail after a minute or so of clicking buttons and need to reset the UI via the 'fix chat' verb.

The previous genetics UI was perfectly functional for me. It's only the recent one I am having trouble with. I needed to install microsoft edge in order to prevent popup failure about a year ago and have had no trouble until today. I have yet to see issues with other popups.

**Comment by nemvar** [comment]
What kind of issue? Does it not pop up at all? Can you not interact with it? Is it pure white?

**Comment by Kingtrin** [comment]
The popup always shows up when clicking the console and varies between full white, gray, and black. I cannot interact with it but the rest of the game is perfectly normal. I get no error messages. The loading symbol appears on my cursor when I hover over it but it never loads on its own.

I forgot to mention but this also happens to me on some, but not all, round end reports. Same problem entirely. Reconnecting does not fix. I need to force reload the chat.

**Comment by PKPenguin321** [comment]
This sounds like it may be an issue unique to you. Post BYOND version, OS, etc

**Comment by Kingtrin** [comment]
Yea I couldnt find anyone experiencing this either despite looking for ages.

I'm on an asus laptop - G74Sx. This has a craptacular gtx 560m thats soldered in. 
8gigs of ram
i7 2630QM, 2 GHz, 8 cores
Win 7 Home Premium 64 bit v6.1
Byond version is 512.1484 (current latest)
Byond is installed into E:/ and not C:/

I found the temp solution mentioned earlier from MSO himself in this thread: http://www.byond.com/forum/post/2286804

This is related to internet explorer which explains why my problem went away when I got edge, but now its cropped up again. For people who have working interfaces (can click links, stable popups) whats the settings and version of your IE/edge?

**Comment by PKPenguin321** [comment]
I'd suggest uninstalling BYOND and then rolling back a build or two from here: http://www.byond.com/download/build/

**Comment by Kingtrin** [comment]
uninstalled and went back to 1484 but still getting the exact same issues.

**Comment by Time-Green** [comment]
An image would really help

**Comment by Kingtrin** [comment]
![Screenshot_4](https://user-images.githubusercontent.com/22778308/65398244-176dbc80-dd84-11e9-9e57-cfb45820e785.png)
![Screenshot_5](https://user-images.githubusercontent.com/22778308/65398245-19d01680-dd84-11e9-9389-12f55413f92a.png)

Funnily enough I was able to play a normal geneticist round only for the round-end screen to bug out. Only shows gray and white screens, so I'm guessing I just imagined the black one. Seems to happen at random actually as previous round-end reports have varied between showing normally and bugging out.

Regardless, this doesn't happen with any other popups.
